### PR TITLE
Stapler: Misc updates

### PR DESCRIPTION
**Pull request by lynnerang**
- Adds generic mobile option for platform
- Conditionally renders keys and heading level based on unassigned or 'web' platform
- Converts default option for heading level to 'unassigned' rather than a mobile setting

### CODE REVIEW

File: stapler/src/views/AriaLabels.svelte [hunk 1]
```diff
@@ -43,7 +44,7 @@
       a11yField={true}
     />
   {:else if roleType === 'interaction' || role === 'heading'}
-    {#if role === 'heading'}
+    {#if role === 'heading' && ['web', 'unassigned'].includes(platform)}
```
Comment (gblakeman) [inline stapler/src/views/AriaLabels.svelte:47]: Up to you, but I kinda like parens here for readability:

```js
{#if (role === 'heading') && ['web', 'unassigned'].includes(platform)}
```

File: stapler/src/views/ComponentData.svelte [hunk 2]
```diff
@@ -28,6 +28,8 @@
   export let resetValue = false;
   export let isAllTab;
 
+  $: console.log(item.platform)
```
Comment (gblakeman) [inline stapler/src/views/ComponentData.svelte:31]: Erroneous log?

**Review by gblakeman — APPROVED** [review]
🥳🎈

A couple small things!

File: stapler/src/views/ComponentData.svelte (hunk 2 above)
Comment (lynnerang) [reply stapler/src/views/ComponentData.svelte:31]: oh yeah, thanks!

**Pull request merged**